lib/sparql_to_sw_solr/instance_topic_fields.rb

```diff
@@ -0,0 +1,38 @@
+module SparqlToSwSolr
+  class InstanceSolrDoc
+    module InstanceTopicFields
```

Comment:
I would name this module (and file) `subject_fields`, for a few reasons:

1.  these topic fields are a subset of the "subject" fields - see https://consul.stanford.edu/display/LD4P/Bibframe+2.0+to+Solr+Mapping

2.  the values of these fields comes from the Work, not the Instance, so I would drop "instance" from the module name